TIKEBook_Lesson2.ipynb

```diff
@@ -0,0 +1,593 @@
+{
```

Comment:
<p>Link to the website you found this on (the TESS Archive Manual?)</p><p> </p><p>We might also want to move this further down, where we discuss what each of these values corresponds to.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Added and moved. </p>